File: source/main/java/com/xeomar/annex/PauseTask.java
package com.xeomar.annex;

import java.util.List;

public class PauseTask extends AnnexTask {

	public PauseTask( List<String> parameters ) {
		super( UpdateFlag.PAUSE, parameters );
	}

	@Override
	public TaskResult execute() throws Exception {
		Thread.sleep( Long.parseLong( getParameters().get( 0 ) ) );
		return new TaskResult( TaskStatus.SUCCESS, "success" );
	}

}

File: source/main/java/com/xeomar/annex/PauseTask.kt
package com.xeomar.annex

class PauseTask(parameters: List<String>) : AnnexTask(command, parameters) {

	override fun execute(): TaskResult {
		Thread.sleep(parameters[0].toLong())
		return TaskResult(TaskStatus.SUCCESS, "success")
	}

	companion object {
		val command = UpdateFlag.PAUSE
	}

}
